[2023-07-22T05:54:12.526+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table scheduled__2023-07-15T00:00:00+00:00 [queued]>
[2023-07-22T05:54:12.558+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table scheduled__2023-07-15T00:00:00+00:00 [queued]>
[2023-07-22T05:54:12.567+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-07-22T05:54:12.575+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-07-22T05:54:12.575+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-07-22T05:54:12.596+0000] {taskinstance.py:1383} INFO - Executing <Task(PostgresOperator): create_table> on 2023-07-15 00:00:00+00:00
[2023-07-22T05:54:12.671+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'create_table', 'scheduled__2023-07-15T00:00:00+00:00', '--job-id', '42', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmp4u2kvz47']
[2023-07-22T05:54:12.679+0000] {standard_task_runner.py:83} INFO - Job 42: Subtask create_table
[2023-07-22T05:54:12.649+0000] {standard_task_runner.py:55} INFO - Started process 5159 to run task
[2023-07-22T05:54:13.239+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.create_table scheduled__2023-07-15T00:00:00+00:00 [running]> on host ef71d3f52879
[2023-07-22T05:54:13.517+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2023-07-15T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-07-15T00:00:00+00:00
[2023-07-22T05:54:13.568+0000] {base.py:71} INFO - Using connection ID 'postgres' for task execution.
[2023-07-22T05:54:13.621+0000] {sql.py:315} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS users (
                firstname TEXT NOT NULL,
                lastname TEXT NOT NULL,
                country TEXT NOT NULL,
                username TEXT NOT NULL,
                password TEXT NOT NULL,
                email TEXT NOT NULL
            );
        , parameters: None
[2023-07-22T05:54:13.628+0000] {postgres.py:96} INFO - NOTICE:  relation "users" already exists, skipping

[2023-07-22T05:54:13.707+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=create_table, execution_date=20230715T000000, start_date=20230722T055412, end_date=20230722T055413
[2023-07-22T05:54:14.026+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-07-22T05:54:14.566+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-07-22T05:56:54.806+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table scheduled__2023-07-15T00:00:00+00:00 [queued]>
[2023-07-22T05:56:54.814+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table scheduled__2023-07-15T00:00:00+00:00 [queued]>
[2023-07-22T05:56:54.815+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-07-22T05:56:54.815+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-07-22T05:56:54.815+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-07-22T05:56:54.843+0000] {taskinstance.py:1383} INFO - Executing <Task(PostgresOperator): create_table> on 2023-07-15 00:00:00+00:00
[2023-07-22T05:56:54.856+0000] {standard_task_runner.py:55} INFO - Started process 5449 to run task
[2023-07-22T05:56:54.878+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'create_table', 'scheduled__2023-07-15T00:00:00+00:00', '--job-id', '99', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmpdh_0o2us']
[2023-07-22T05:56:54.880+0000] {standard_task_runner.py:83} INFO - Job 99: Subtask create_table
[2023-07-22T05:56:55.121+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.create_table scheduled__2023-07-15T00:00:00+00:00 [running]> on host ef71d3f52879
[2023-07-22T05:56:55.294+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2023-07-15T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-07-15T00:00:00+00:00
[2023-07-22T05:56:55.317+0000] {base.py:71} INFO - Using connection ID 'postgres' for task execution.
[2023-07-22T05:56:55.328+0000] {sql.py:315} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS users (
                firstname TEXT NOT NULL,
                lastname TEXT NOT NULL,
                country TEXT NOT NULL,
                username TEXT NOT NULL,
                password TEXT NOT NULL,
                email TEXT NOT NULL
            );
        , parameters: None
[2023-07-22T05:56:55.337+0000] {postgres.py:96} INFO - NOTICE:  relation "users" already exists, skipping

[2023-07-22T05:56:55.358+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=create_table, execution_date=20230715T000000, start_date=20230722T055654, end_date=20230722T055655
[2023-07-22T05:56:55.450+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-07-22T05:56:55.557+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
